### DISCUSSION TITLE
[doris-2.0.3-rc06] 在两个集群之间使用catalog 迁移数据 报错  使用的OpenJdk

### DISCUSSION THREAD

**Discussion by wangxiaopanGit** [apache/doris#29281] [A - General / Q&A] [answered]
SQL 错误 [1105] [HY000]: errCode = 2, detailMessage =[INTERNAL_ERROR]GetJniExceptionMsg meet error, query=SELECT `business_date`, `corp_id`, `plate_number`, `accident_sn`, `source`, `source_id`, `id`, `area_code`, `area_name`, `corp_name`, `org_id`, `org_name`, `org_code`, `org_long_code`, `org_long_name`, `line_id`, `line_code`, `line_name`, `vehicle_id`, `vehicle_code`, `empl_id`, `empl_name`, `work_no`, `accident_time`, `accident_content`, `accident_address`, `accident_status`, `accident_type`, `accident_process_type`, `accident_duty_type`, `close_remark`, `close_time`, `safety_name`, `duty_safety_name`, `accident_person_cost`, `accident_vehicle_cost`, `accident_asset_cost`, `accident_in_amount`, `accident_out_amount`, `accident_inout_amount`, `close_amount`, `damage_amount`, `longitude`, `latitude`, `claim_situation`, `creator_id`, `creator_name`, `create_time`, `modifier_name`, `modify_id`, `modify_time`, `operator_id`, `operator_name`, `process_time` FROM test WHERE (business_date = '2023-11-20'), msg=[INTERNAL_ERROR]OutOfMemoryError: Java heap space

	0#  doris::JniUtil::GetJniExceptionMsg(JNIEnv_*, bool, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) at /root/src/doris-2.0/be/src/util/jni-util.h:110
	1#  doris::vectorized::JdbcConnector::query() at /var/local/ldb-toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/basic_string.h:187
	2#  doris::vectorized::NewJdbcScanner::open(doris::RuntimeState*) at /root/src/doris-2.0/be/src/common/status.h:442
	3#  doris::vectorized::ScannerScheduler::_scanner_scan(doris::vectorized::ScannerScheduler*, doris::vectorized::ScannerContext*, std::shared_ptr<doris::vectorized::VScanner>) at /root/src/doris-2.0/be/src/common/status.h:354
	4#  std::_Function_handler<void (), doris::vectorized::ScannerScheduler::_schedule_scanners(doris::vectorized::ScannerContext*)::$_1::operator()() const::{lambda()#4}>::_M_invoke(std::_Any_data const&) at /var/local/ldb-toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/shared_ptr_base.h:701
	5#  doris::WorkThreadPool<true>::work_thread(int) at /var/local/ldb-toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/atomic_base.h:646
	6#  execute_native_thread_routine at /data/gcc-11.1.0/build/x86_64-pc-linux-gnu/libstdc++-v3/include/bits/unique_ptr.h:85
	7#  start_thread
	8#  __clone

**Comment by LemonLiTree** [comment, marked as answer]
不建议这么做，推荐backup restore

**Reply by wangxiaopanGit** [reply]
copy that
